src/pages/AdminDashboard.tsx

```
@@ -197,212 +202,216 @@ const AdminDashboard: React.FC = () => {
+                onChange={e => setTimeRange(e.target.value)}
+                className='input text-sm'
+              >
+                <option value='24h'>Last 24 hours</option>
+                <option value='7d'>Last 7 days</option>
+                <option value='30d'>Last 30 days</option>
+                <option value='90d'>Last 90 days</option>
+              </select>
+              <button
+                onClick={exportData}
+                className='btn-secondary btn-md flex items-center space-x-2'
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)